src/components/header/index.js

```diff
@@ -72,8 +72,8 @@ export const Header = () => {
               {/* HIC LOGO */}
               {true && (
                 <img
-                  src={`https://teia.art/logos/${context.theme}/${context.logo}`}
-                  alt="teia"
+                  src={`https://raw.githubusercontent.com/teia-community/teia-logos/main/dist/logos/${context.theme}/${context.logo}`}
```

Comment:
would it make sense to move this url into an env variable as well? or just store the `https://raw.githubusercontent.com/teia-community/teia-logos/main/dist/` part as env?

Reply:
Makes sense! so using on env by removing 'logo.json' right!
Doing it ;)

Reply:
Done in a39d78e2e8372b2c9a7aae1a2e7de852e10e0521